package.json

```diff
@@ -51,80 +51,78 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@bull-board/api": "3.6.0",
+    "@bull-board/api": "3.7.0",
     "@bull-board/express": "3.7.0",
-    "@elastic/elasticsearch": "7.11.0",
-    "@elastic/elasticsearch-mock": "0.3.0",
-    "@wordpress/wordcount": "2.15.0",
```

Comment:
I told you the wrong thing, I forgot the status service uses this.  We better leave this in.